kpatch-build/create-diff-object.c

```diff
@@ -302,7 +310,9 @@ static int kpatch_warn_only_change(struct section *sec)
 				continue;
 			if (!strncmp(rela->sym->name, "__warned.", 9))
 				continue;
-			if (!strncmp(rela->sym->name, "warn_slowpath_", 14)) {
+			if (!strncmp(rela->sym->name, "warn_slowpath_", 14) ||
+			   (!strncmp(rela->sym->name, "__might_sleep", 14)) ||
+			   (!strncmp(rela->sym->name, "___might_sleep", 15))) {
```

Comment:
I think just a strcmp compare will suffice since they're not substrings.  Otherwise the patch looks good to me.